### PR TITLE
feature/jm-5969- failed to attend letters

### PR DESCRIPTION
**Pull request by hari-vajja**
### JIRA link ###
https://centralgovernmentcgi.atlassian.net/browse/JM-5969

### Change description ###

Added functionality for generating Failed to Attend Letters through the documents section and through a juror's record from the attendance tab.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: server/routes/documents/failed-to-attend/failed-to-attend.controller.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+          auth: req.session.authentication,
+          jwt: req.session.authToken,
+          data: { ...req.body },
+          error: (typeof err.error !== 'undefined') ? err.error : err.toString(),
+        });
+
+        return res.render('_errors/generic.njk');
+      }
+
+    };
```
Comment (ricardofreitasrocha) [inline server/routes/documents/failed-to-attend/failed-to-attend.controller.js:126]: is any of this necessary with a en empty `try{}` block?

**Review by ricardofreitasrocha — CHANGES_REQUESTED** [review]

File: server/routes/documents/failed-to-attend/failed-to-attend.controller.js (hunk 1 above)
Comment (hari-vajja) [reply server/routes/documents/failed-to-attend/failed-to-attend.controller.js]: Removed

File: server/routes/documents/failed-to-attend/failed-to-attend.controller.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
+(function() {
+  'use strict';
+
+  const _ = require('lodash')
+    , urljoin = require('url-join')
+    , { reissueLetterDAO } = require('../../../objects/documents')
+    , modUtils = require('../../../lib/mod-utils')
+    , { isCourtUser } = require('../../../components/auth/user-type');
```
[the remark this answers is not in the document]
Comment (ricardofreitasrocha) [reply server/routes/documents/failed-to-attend/failed-to-attend.controller.js:8]: @hari-vajja missing these changes still

**Review by ricardofreitasrocha — CHANGES_REQUESTED** [review]

File: server/routes/documents/failed-to-attend/failed-to-attend.controller.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (hari-vajja) [reply server/routes/documents/failed-to-attend/failed-to-attend.controller.js]: Done

**Review by ricardofreitasrocha — APPROVED** [review]

**Pull request merged**